### PR TITLE
hotfix + spec: Added missing test cases for AWS CloudTrail plugins | Fixed syntax errors

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/cloudtrail/cloudtrailObjectLock.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = {
             }
 
             if (!describeTrails.data.length) {
-                helpers.addResult(results, 0, 'No trails found', region);
+                helpers.addResult(results, 2, 'CloudTrails is not enabled', region);
```
Comment (giorod3) [inline plugins/aws/cloudtrail/cloudtrailObjectLock.js:32]:
```suggestion
                helpers.addResult(results, 0, 'No trails found', region);
```

Comment (giorod3) [reply plugins/aws/cloudtrail/cloudtrailObjectLock.js:32]: I am going to revert this

Comment (giorod3) [inline plugins/aws/cloudtrail/cloudtrailObjectLock.js]: actually, it will mess up the tests, can you revert this to No trails found, it sounds better.

**Review by giorod3 — CHANGES_REQUESTED** [review]
just the one change and this is ready to go

File: plugins/aws/cloudtrail/cloudtrailObjectLock.js (hunk 1 above)
Comment (AkhtarAmir) [reply plugins/aws/cloudtrail/cloudtrailObjectLock.js]: I had changed the test as well. The thing is that in all other CloudTrail plugins, we are displaying CloudTrail is not enabled. So I changed it to make it consistent. Do you still want me to change it back?

**Review by giorod3 — APPROVED** [review]

**Review by dipsubha06 — APPROVED** [review]

**Pull request merged**